Adafruit_MLX90614.h

```diff
@@ -53,11 +53,17 @@ class Adafruit_MLX90614 {
   double readAmbientTempC(void);
   double readObjectTempF(void);
   double readAmbientTempF(void);
+  uint16_t readEmmissivityReg(void);
```

Comment:
please rename functions to `Emissivity` (one m)

Reply:
Foo!   i kept getting that wrong and also emissitivity...  will fix.  thanks for catching.

Reply:
Is fixed now.  I didn't update the version number in the library file, I figure that was up to you.

do you need a new PR?

Reply:
nope!